jbpm-flow-builder/src/test/java/org/jbpm/process/workitem/WorkDefinitionImplTest.java

```diff
@@ -122,7 +122,7 @@ public void testParameterValuesWithEnumsOnly() throws Exception {
             if (entry.getKey().equals("param1")) {
                 EnumDataType paramEnum = (EnumDataType) entry.getValue();
                 assertEquals("org.jbpm.process.workitem.enums.AnimalsEnum", paramEnum.getClassName());
-                Map<String, Object> paramValuesMap = paramEnum.getValueMap();
+                Map<String, Object> paramValuesMap = paramEnum.getValueMap(null);
```

Comment:
Agree. I think the drools change for this should have added a getValueMap method that allows to pass CL instead of replacing existing no param one.